partner_event/hooks.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 David Vidal
+# License AGPL-3 - See https://www.gnu.org/licenses/agpl-3.0.html
+from odoo import api, SUPERUSER_ID
+
+
+def post_init_hook(cr, registry):
+    """Preload proper attendee partner for existing registrations using
+       the same rules the module does"""
+    with api.Environment.manage():
+        env = api.Environment(cr, SUPERUSER_ID, {})
+        attendees = env['event.registration'].search([
```

Comment:
Do a `read_group` by email for searching only once each email

Reply:
Change done